analyzers/tests/SonarAnalyzer.UnitTest/Rules/ClassAndMethodNameTest.cs

```diff
@@ -49,7 +49,7 @@ public class ClassAndMethodNameTest
         [TestMethod]
         [TestCategory("Rule")]
         public void ClassName_InTestProject_CS() =>
-            Verifier.VerifyAnalyzerInTest(@"TestCases\ClassName.Tests.cs", new CS.ClassAndMethodName(), ParseOptionsHelper.FromCSharp8);
+            Verifier.VerifyAnalyzer(@"TestCases\ClassName.Tests.cs", new CS.ClassAndMethodName(), ParseOptionsHelper.FromCSharp8, NuGetMetadataReference.MSTestTestFrameworkV1);
```

Comment:
I suggest to use `TestHelper.ProjectTypeReference(ProjectType.Test)`, to keep the decision about the reference centralized

Reply:
That would be parametrized indirection (unneeded complexity). In this specific case, we're testing test-only scenario that doesn't apply to product code and we don't need to go through the specific logic inside `ProjectTypeReference`